core/sr-primitives/src/traits.rs

```diff
@@ -303,6 +303,47 @@ pub trait CheckedConversion {
 }
 impl<T: Sized> CheckedConversion for T {}
 
+/// Multiply and devide by a number that isn't necessarily the same type. Basically just the same
```

Comment:
```suggestion
/// Multiply and divide by a number that isn't necessarily the same type. Basically just the same
```

Reply:
It feels a little unfortunate we have a SCALE codec and a Scale trait, but that may just be me.